### ISSUE TITLE
Notification only works the first time

### ISSUE THREAD

**Issue by wussuphaters** [dnpwwo/Domoticz-Google-Plugin#39]
Hi,

When my Lua script is triggered and I send text to my Home Mini, i hear the beep followed by my text. But during the following script calls, I only hear the beep, or the beep and, faintly, the first letter of my notification. When I restart Domoticz it works fine again once, then doesn't. Any idea why ? I get no error logs even though they all logs are enabled for this plugin.

**Comment by jotheman0303** [comment]
Yep, same issue here... Is this project still being supported? I see little updates on it lately

**Comment by atmfrenki** [comment]
Same here, looks like the MP3 file created gets deleted to fast, domoticz log says xxxxx.MP3 does not exist

**Comment by atmfrenki** [comment]
> > The same problem here, my Pi is little bit overloaded, so i blame that as the issue. added/changed these lines. Its better not perfect every time. ![afbeelding](https://user-images.githubusercontent.com/11648297/158446459-272f7523-ab4f-4840-98ec-069bd9f8146f.png)
> 
> That looks like a good workaround at least. Can you explain a bit on where this should be changed (the file and its location)? Would help me out greatly. Thanks in advance!

It is in the plugin.py file in your plugin folder of the Domoticz-Google-Plugin.
/domoticz/plugins/Domoticz-Google-Plugin

The 1st (time.sleep(2.0) is a blank line, so add it completely
The 2ns is an adjustment of an existing value: time.sleep(1.0) -> time.sleep(4.0)

I'm testing this out. Thanks [Xenomes](https://github.com/Xenomes)

**Comment by jotheman0303** [comment]
> > > The same problem here, my Pi is little bit overloaded, so i blame that as the issue. added/changed these lines. Its better not perfect every time. ![afbeelding](https://user-images.githubusercontent.com/11648297/158446459-272f7523-ab4f-4840-98ec-069bd9f8146f.png)
> > 
> > 
> > That looks like a good workaround at least. Can you explain a bit on where this should be changed (the file and its location)? Would help me out greatly. Thanks in advance!
> 
> It is in the plugin.py file in your plugin folder of the Domoticz-Google-Plugin. /domoticz/plugins/Domoticz-Google-Plugin
> 
> The 1st (time.sleep(2.0) is a blank line, so add it completely The 2nd is an adjustment of an existing value: time.sleep(1.0) -> time.sleep(4.0)
> 
> I'm testing this out. Thanks [Xenomes](https://github.com/Xenomes)
> 
> Update: it works pretty good. I installed pychromecast 9.2.1

This works great indeed! Put mine on 2/5 and its great, no issues at all anymore! Thanks everyone!
